src/DGtal/dec/DiscreteExteriorCalculus.h

```diff
@@ -400,6 +401,17 @@ namespace DGtal
     LinearOperator<Self, 0, duality, 0, duality>
     laplace() const;
 
+    /**
+     * Convolutional Laplace operator from duality 0-forms to duality 0-forms. See \ref secHowToHeatLaplace.
+     * @param h the grid step
+     * @param t the time parameter for the convolution
+     * @param cut_locus the cut locus for for the integration. Generaly setted to \f$ \log( - \log( t ) + 1 ) + 2 \f$
```

Comment:
for for -> for
setted->set

the "cut locus" is not very clear... please rephrase.